templates/adherent_profile/activity/_instances.html.twig

```diff
@@ -0,0 +1,31 @@
+<h1 class="text--medium-large">Mes instances</h1>
+
+<div class="l__col">
+    {% if app.user.territorialCouncilMembership %}
```

Comment:
IMO we can avoid this check as we already test it in `activity.html.twig` template

Reply:
indeed, removed